### ISSUE TITLE
Binary download : assembly not strong named

### ISSUE THREAD

**Issue by dahall** [dahall/TaskScheduler#360]
Hi, in the binary download of the TaskScheduler assembly, the assembly is not signed.  Probably it is a good idea distribute it as a strong named assembly.

Also, the source code contains a (the ?) signing key file.

So, for now I made a compilation of my own and signed it with this key, but I believe you better make available a signed binary.

Joris Spriet

Originally posted: 2011-10-04T08:07:51

**Comment by dahall** [comment]
Thank you for your suggestion. The reason the key is there is specifically so it can be compiled and signed by those that need it. When I released it a couple of years ago signed, I had some big backlash from the community.



Originally posted: 2011-10-04T12:53:11

**Comment by dahall** [comment]
If you can elaborate I would like to know what requirements you have for strongly named assemblies so I can make a more informed decision. Also, what problems have you run into with strongly named assemblies in the past and how have you worked around them?



Originally posted: 2011-10-04T12:57:31

**Comment by dahall** [comment]
At my company, we sign all our assemblies.  It serves as a sign to our customers the assembly can be trusted - and it was produced on our build server.



As a consequence, we only can use third-party assemblies that are signed as well.  If a third-party component is not available signed,



we basically don't use it.  For TaskScheduler, as the source contained a snk, I compiled and signed it myself.



The library was just what I needed, and, okay, so I cheated here, but that is why I posted this remark...







Joris.











Originally posted: 2011-10-04T17:48:52
